airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.java

```diff
@@ -745,9 +744,9 @@ public void testCdcOrdering_insertAfterDelete() throws Exception {
     destinationHandler.execute(sql);
 
     verifyRecordCounts(
-        1,
+        2,
         dumpRawTableRecords(streamId),
-        1,
+        0,
```

Comment:
@edgao I'm working though updating all the test fixtures to live in a world where we don't dedupe the raw table, and bumping expectedRawRecords from 1 to 2 is right, but why would the expectedFinalRecords change to 0 here?

Reply:
that... sounds like a legit bug in the sql? not deduping the raw table shouldn't have any effect on the final table. This test is exercising a pretty specific out-of-order sequencing (see its javadoc).

I think we need to modify the CDC delete statement to also filter for `raw_cursor > final_cursor`? Since previously we would just dedup out the older deletion record

(I continue to regret writing some of these tests as record counts instead of actually having expected records)